wifiphisher/pywifiphisher.py

```diff
@@ -135,6 +141,21 @@ def check_args(args):
             '-' +
             W +
             '] Pre-shared key must be between 8 and 63 printable characters.')
+    if args.handshakecapture and not args.presharedkey:
+        sys.exit(
+            '[' +
+            R +
+            '-' +
+            W +
+            '] handshake-capture should be used with the presharekey option.')
```

Comment:
Yes. I will remove this check since orginally I misundertand this feature and check the users ' psk input.